dracclient/resources/raid.py

```diff
@@ -259,6 +259,11 @@ def _parse_drac_physical_disk(self,
                                                        uri)
         drac_bus_protocol = self._get_physical_disk_attr(drac_disk,
                                                          'BusProtocol', uri)
+        if uri == uris.DCIM_PCIeSSDView:
+            drac_bus_number = self._get_physical_disk_attr(drac_disk,
```

Comment:
Can you call .lower() on the bus number if it's not None?  It would be great if you could modify the controller parsing so that it does the same as well for consistency.

Reply:
Actually, I take that back.  The iDRAC appears to return upper case for the bus ID, so let's call .upper() in both places so we don't potentially break clients.

Reply:
Actually we have handle this in Jetpack/assign_role by doing the bus id to the lowercase because while creating device-by-path name for nmve drives we need bus id as lower(). But anyway we can also make it upper() here. Consider it done

Reply:
Yes, I know.  I just want to make sure that the bus ID is returned from the python-dracclient API in the same case in both places where we return it (controllers and physical disks)

Reply:
Thanks. will update in an upstream patch.